docs/toolkit/ansible-avalanche-collection/reference/roles/avalanche-node.md

```diff
@@ -27,6 +27,7 @@ This Ansible role allows to bootstrap Avalanche nodes:
 | `avalanchego_group`                   | The group of the user that will run the AvalancheGo Linux service                                                                                                                                                                                                     | `avalanche`                                                                                            |
 | `avalanchego_http_host`               | [--http-host](https://docs.avax.network/build/references/avalanchego-config-flags#--http-host-string) argument                                                                                                                                                        | `127.0.0.1`                                                                                            |
 | `avalanchego_http_port`               | [--http-port](https://docs.avax.network/build/references/avalanchego-config-flags#--http-port-int) argument                                                                                                                                                           | `9650`                                                                                                 |
+| `avalanchego_http_allowed_hosts`      | [--http-allowed-hosts](https://docs.avax.network/build/references/avalanchego-config-flags#--http-allowed-hosts-string) argument                                                                                                                                      | `localhost`                                                                                            |
```

Comment:
The field does not exist in the Avalanche docs. We should open a PR there.

Reply:
Yes I figured they will implement it afterwards but it looks like they still have not done it

Reply:
https://github.com/ava-labs/avalanche-docs/issues/1434